### PR TITLE
Translations added successfully for these three languages : Amheric ,French, Khmer

### PR DESCRIPTION
**Pull request by Art-Ndiema**
## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes the ticket number in the format `OHRI-123 My PR title`.
- [ ] My work includes tests or is validated by existing tests.

## Screenshots

Screenshot 2024-07-01 at 10 47 22 AM
Screenshot 2024-07-01 at 10 48 21 AM
Screenshot 2024-07-01 at 10 52 38 AM

### CODE REVIEW

**Review by CynthiaKamau — COMMENTED** [review]
Thanks @Art-Ndiema , please undo the changes on the package.json and yarn.lock file since we use next `openmrs` version

**Review by CynthiaKamau — APPROVED** [review]

File: packages/esm-cervical-cancer-app/translations/km.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "cacxSummary": "CaCx Summary"
```
Comment (hadijahkyampeire) [inline packages/esm-cervical-cancer-app/translations/km.json:2]:
```suggestion
  "cacxSummary": "CaCx Summary"
  "cacxSummary": "cacx សង្ខេប
```

File: packages/esm-cervical-cancer-app/translations/fr.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
+  "cacxSummary": "CaCx Summary"
```
Comment (hadijahkyampeire) [inline packages/esm-cervical-cancer-app/translations/fr.json:2]:
```suggestion
  "cacxSummary": "CaCx Summary"
  "cacxSummary": "Résumé du cacx"
```

File: packages/esm-cervical-cancer-app/translations/am.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+  "cacxSummary": "CaCx Summary"
```
Comment (hadijahkyampeire) [inline packages/esm-cervical-cancer-app/translations/am.json:2]:
```suggestion
  "cacxSummary": "CaCx Summary"
  "cacxSummary": "cacx ማጠቃለያ
"
```

File: packages/esm-ohri-core-app/translations/km.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
+  "programmes": "Programmes"
```
Comment (hadijahkyampeire) [inline packages/esm-ohri-core-app/translations/km.json:2]:
```suggestion
  "programmes": "Programmes"
  "programmes": "កម្មវិធី"
```

File: packages/esm-ohri-core-app/translations/am.json [hunk 5]
```diff
@@ -0,0 +1,3 @@
+{
+  "programmes": "Programmes"
```
Comment (hadijahkyampeire) [inline packages/esm-ohri-core-app/translations/am.json:2]:
```suggestion
  "programmes": "Programmes"
  "programmes": "ፕሮግራሞች"
```

File: packages/esm-tb-app/translations/fr.json [hunk 6]
```diff
@@ -0,0 +1,8 @@
+{
+  "activeDrCases": "Active DR Cases",
```
Comment (hadijahkyampeire) [inline packages/esm-tb-app/translations/fr.json:2]: Lets translate these, for example the first value should be `Cas de reprise après sinistre actifs`

**Review by hadijahkyampeire — CHANGES_REQUESTED** [review]
Thanks @Art-Ndiema but I think you missed translating some values

File: packages/esm-cervical-cancer-app/translations/am.json [hunk 7]
```diff
@@ -0,0 +1,5 @@
...
```
Comment (hadijahkyampeire) [inline packages/esm-cervical-cancer-app/translations/am.json:2]: You don't need these other English translations, all you need is the same key.

File: packages/esm-cervical-cancer-app/translations/fr.json [hunk 8]
```diff
@@ -0,0 +1,4 @@
...
```
Comment (hadijahkyampeire) [inline packages/esm-cervical-cancer-app/translations/fr.json:2]: ditto

File: packages/esm-cervical-cancer-app/translations/km.json [hunk 9]
```diff
@@ -0,0 +1,4 @@
...
```
Comment (hadijahkyampeire) [inline packages/esm-cervical-cancer-app/translations/km.json:2]: ditto

**Review by hadijahkyampeire — COMMENTED** [review]
Just minor cleanups

**Comment by GKMuthee** [comment]
@hadijahkyampeire @Art-Ndiema has Amharic been implemented on this PR? 
Regarding your question on other languages: Portuguese and Swahili.
cc: @eudson @ebambo

**Comment by hadijahkyampeire** [comment]
Thanks @GKMuthee , Amharic is already added, @Art-Ndiema will add Swahili and Portuguese

**Review by hadijahkyampeire — APPROVED** [review]
Thanks @Art-Ndiema , let's merge this

**Pull request merged**